documentation/docs/specs/revocation_bitmap_2022.md

```diff
@@ -0,0 +1,196 @@
+
+The following data URL decodes to a bitmap of length 16384 where all indices are revoked:
+
+`"data:application/octet-stream;base64,ZUp6dHhERVJBQ0FNQkxESEFWS1lXZkN2Q3E0MmFESmtyMlNrM0ROckFLQ2RBQUFBQUFBQTMzbGhHZm9q"`
+
+## Rationale
+
+This section describes the rationale behind some of the design decisions of this specification.
+
+### Compression
+
+Considering that messages published to the Tangle cannot exceed [32 KiB](https://github.com/iotaledger/tips/blob/main/tips/TIP-0006/tip-0006.md#message-validation) in size, and that larger messages have increased requirements, the use of compression was assessed.
+
+ZLIB [[RFC 1950](https://datatracker.ietf.org/doc/html/rfc1950)] was chosen for having a free and open source software licence and being one of the most widely used compression schemes, which enhances the accessibility of this specification. Some other assessed algorithms had only marginally better compression ratios but far fewer existing implementations in different programming languages.
+
+### Compressed Bitstring vs. Roaring Bitmap
+
+Because of their space efficiency, a roaring bitmap is preferred for representing a bitmap in-memory. To avoid the dependency on roaring bitmap, we considered to use a compressed bitstring as the serialization format. However, serialization of such a bitstring is 2-3x slower compared to roaring's serialization format, which becomes an issue on resource-constrained devices (e.g. smartphones) or in web browsers.
+
+### Comparison to `RevocationList2020`
+
+The [RevocationList2020 specification](https://w3c-ccg.github.io/vc-status-rl-2020/) describes a revocation mechanism using a verifiable credential that contains a bitmap, similar to the `RevocationBitmap2022` approach. The credential is hosted outside of the DID document and the verifier thus needs to fetch it from an external resource, likely one controlled by the issuer. This has privacy implications as the issuer can track where a fetch request for the credential came from and infering who the credential was verified by and for what purpose. The issuer can also potentially infer which credential was checked. Because `RevocationBitmap2022` is embedded in the DID document, which can be obtained without the issuer's knowledge, this approach does not suffer from those shortcomings. See also the [privacy considerations](#privacy-considerations).
+
+A downside of embedding the revocation list in the DID document is that storage in a distributed ledger (DLT) is usually more expensive than storage outside of it. The DLT might also impose message size limitations, capping the total number of revocations that can be done (see also [compression](#compression)).
+
+### Privacy Considerations
+
+Because the revocation bitmap is embedded in the DID document, and thus available without contacting the issuer directly, the issuer cannot correlate how a holder uses their credential. Because of that, it is not necessary for the bitmap to have a minimum size.
```

Comment:
Remove reference to minimum size, see previous comment which suggests adding a separate paragraph for it.

```suggestion
Because the revocation bitmap is embedded in the DID document, and thus available without contacting the issuer directly, the issuer cannot correlate how a holder uses their credential.
```